server/src/main/java/org/candlepin/policy/js/compliance/hash/Hasher.java

```diff
@@ -28,19 +32,28 @@
     private static Logger log = LoggerFactory.getLogger(Hasher.class);
     private StringBuilder sink;
 
+    // Trivia: the second seed is the seventh Mersenne prime
+    private static final HashFunction HASH_FUNCTION = Hashing.concatenating(
+        Hashing.murmur3_128(),
+        Hashing.murmur3_128(524287)
+    );
+
     public Hasher() {
         sink = new StringBuilder();
     }
 
     /**
-     * Products an SHA256 hash of anything that was put into this hasher.
+     * Produces two concatenated Murmur3 hashes of anything that was put into this hasher to produce a 256
+     * bit digest.  We concatenate two 128 bit digests for legacy reasons: previously we used SHA256 which
```

Comment:
Would it be faster to concatenate the same hash twice (i.e. 

```java
HASH_FUNCTION = Hashing.murmur3_128();
//...
return c.toString() + c.toString()
```
)? Or something similar?

Reply:
Slightly faster, yes.  Probably on the order of another 1 or 2%.  I was basing my approach on [this](https://github.com/google/guava/blob/master/guava/src/com/google/common/hash/Hashing.java#L75).  I think I would like to keep it so that the hash is distributed across 256 bits instead of just 128 doubled up.